jobs/telemetry_batch_view.py

```diff
@@ -1,33 +1,30 @@
 #!/usr/bin/env python
 
+from requests
```

Comment:
Probably supposed to be "import requests".

Reply:
hmm, concerning that I missed that. I'll make sure to run this locally.

Reply:
Annoying that launching locally still retrieves the file from the master Airflow repo. Something else to fix...